Lib/dis.py

```diff
@@ -32,7 +32,8 @@ def _try_compile(source, name):
     return c
 
 def dis(x=None, *, file=None, depth=None):
-    """Disassemble classes, methods, functions, generators, or code.
+    """Disassemble classes, methods, functions, generators,
```

Comment:
Best yet, but I regret dropping code and methods. How about this:

~~~~
"""Disassemble classes, methods, functions, compiled objects, or code.

With no argument, disassemble the last traceback.

Compiled objects currently include generator objects, async generator objects,
and coroutine objects, which store their code in special attributes.
"""
~~~~